agent/agentssh/agentssh.go

```diff
@@ -63,9 +64,10 @@ type Server struct {
 	srv          *ssh.Server
 	x11SocketDir string
 
-	Env        map[string]string
-	AgentToken func() string
-	Manifest   *atomic.Pointer[agentsdk.Manifest]
+	Env              map[string]string
+	AgentToken       func() string
+	Manifest         *atomic.Pointer[agentsdk.Manifest]
+	GetServiceBanner func(ctx context.Context) (codersdk.ServiceBannerConfig, error)
```

Comment:
Suggestion: Drop the "get" to match other methods (e.g. `AgentToken`), see: https://golang.org/doc/effective_go.html#Getters
 
```suggestion
	ServiceBanner func(ctx context.Context) (codersdk.ServiceBannerConfig, error)
```

Reply:
Ahhh very good to know, will change and thank you for the informative link!